src/DotnetDocument/Extensions/MemberDeclarationSyntaxExtensions.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Linq;
+using DotnetDocument.Configuration;
+using DotnetDocument.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace DotnetDocument.Extensions
+{
+    public static class MemberDeclarationSyntaxExtensions
+    {
+        public static bool ShouldDocument(this MemberDeclarationSyntax node, MemberDocumentationOptionsBase options)
+        {
+            var shouldDocument = false;
+
+            foreach (var modifier in options.AccessModifiers)
+            {
+                var modifierArray = modifier.Split(" ");
```

Comment:
@tmonck I cloned your repo to try this out and this logic doesn't seem to work.  For example, when I just had "public" in the access modifiers list in the config, the `if (modifierArray.Length > 1)` statement gets skipped over entirely

Reply:
This is my workaround
```cs
private static readonly List<string> _accessorKeyWords =
[
    "public",
    "internal",
    "private",
    "protected"
];

public static bool ShouldDocument(this MemberDeclarationSyntax node, MemberDocumentationOptionsBase options)
{
    var nodeModifiers = node.Modifiers.Select(m => m.Text)
        .Where(mText=> _accessorKeyWords.Contains(mText));
    return options.AccessModifiers.Any(m=>
    {
        var optionModifier = m.Split(' ');
        return nodeModifiers.All(nodeModifier => optionModifier.Contains(nodeModifier));
    });
}
```
This doesn't allow the user to specify any valid modifiers besides what's in `_accessorKeyWords` so not sure if this works for your implementation